eng/helix/vstest/runtests.cmd

```diff
@@ -36,7 +36,7 @@ REM only puts the explicit filter traits the user provided in
 REM Filter syntax: https://github.com/Microsoft/vstest-docs/blob/master/docs/filter.md
 set NONFLAKY_FILTER="Flaky:All!=true&Flaky:Helix:All!=true&Flaky:Helix:Queue:All!=true&Flaky:Helix:Queue:%HELIX%!=true"
 echo Running non-flaky tests.
-%DOTNET_ROOT%\dotnet vstest %target% --logger:trx --logger:console;verbosity=normal --TestCaseFilter:%NONFLAKY_FILTER%
+%DOTNET_ROOT%\dotnet vstest %target% --logger:trx;LogFileName=test-results.xml --TestCaseFilter:%NONFLAKY_FILTER%
```

Comment:
```suggestion
%DOTNET_ROOT%\dotnet vstest %target% --logger:trx --TestCaseFilter:%NONFLAKY_FILTER%
```

No need to change the file name manually. https://github.com/dotnet/arcade/pull/2446/ implemented support for .trx files, and they will scan for any file named *.trx which is produced in the working directory (or subdirs).